# Question 16
-----

In [1]:
import math

print(f"effect = {math.exp(-0.24411)}")

effect = 0.78340145536302


In [2]:
print(f"common ratio: {1/0.7834}")

common ratio: 1.2764871074802144


The effect $e^{-0.24411}$ tells us that a person with low vitamin C intake is 1.28 times more likely to get cervical cancer.